src/js/requestHandlers.js

```diff
@@ -84,10 +84,14 @@ sjrk.storyTelling.server.handleGetStory = function (request, dataSource, uploade
 
         fluid.transform(response.content, function (block) {
             if (block.blockType === "image") {
```

Comment:
This section is a bit repetitive, and could use a little tidying up. I realize you were attempting to make minimal changes to the code, so please feel free to disregard this suggestion. It might be better to parameterize the imageUrl/mediaUrl until such time as the two values are combined in the UI project (there is a plan for a bit of refactoring there when we have time). I was thinking something along these lines:

```suggestion
            var urlValueName = "mediaUrl";

            if (block.blockType === "image") {
                urlValueName = "imageUrl";
            }

            if (block[urlValueName]) {
                block[urlValueName] = uploadedFilesHandlerPath + "/" + block[urlValueName];
            }

            return block;
```

Reply:
Thanks Gregor. I think at this time, I will leave the code as is. I want to introduce the minimum amount of new logic and I want it to be as clear as possible. At the cost of some amount of repetition.